static/js/topic_list_data.js

```diff
@@ -33,7 +33,7 @@ export function get_list_info(stream_id, zoomed) {
         const is_topic_muted = muted_topics.is_topic_muted(stream_id, topic_name);
 
         if (!zoomed) {
-            function should_show_topic(topics_selected) {
+            function should_show_topic(no_topics_selected) {
```

Comment:
Let's call this `selected_topic_count`; I read `no_topics_selected` as a boolean at first.